nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -216,24 +215,28 @@ def _is_node_correctable(self, node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
             node, nncf_graph
         )
 
-    def _get_subgraph_data_for_node(self, node: NNCFNode, nncf_graph: NNCFGraph) -> Dict[str, List[str]]:
+    def _find_subgraph_output_ids(self, nncf_graph: NNCFGraph, main_node: NNCFNode) -> List[Tuple[str, int]]:
```

Comment:
Can be these two traversal functions merged into one single traverse from a node with a bias up to collected statistics?

Reply:
It was implemented in `LayerWiseEngine` and can be used in BC.